libraries/monitor/src/logger.js

```diff
@@ -169,6 +179,7 @@ class Logger {
       Fields: fields,
       message, // will be omitted if undefined
       traceId, // will be omitted if undefined
+      requestId, // will be omitted if undefined
```

Comment:
We already don't fit MozLog directly. Our log format is a superset of it so we guarantee fields are there that they want but not that they're the only fields available.

I moved them up a level because otherwise adding them to arbitrary `Fields` would make the schematization of `Fields` confusing and inconsistent. Happy to move them back into `Fields` or into another subfield of the top-level. Whatever makes more sense. I'll document whatever decision we make.

Reply:
The point about schematization makes sense - I hadn't thought of that.  So yeah, let's document them as-is (https://docs.taskcluster.net/docs/manual/design/logs).